requirements.d/development.txt

```diff
@@ -1,4 +1,5 @@
 setuptools
+setuptools_scm
```

Comment:
the one in setup.py will trigger easy_install, this one will do pip install
once pip does PEP-518 this can be moved from both locations to pyproject.toml